[IMP] helpdesk: close_ticket.rst cleanup (#3228)
odoo/documentation: content/applications/services/helpdesk/advanced/close_tickets.rst

======================================
Allow customers to close their tickets
======================================

Allowing customers to close their tickets gives them autonomy and minimize misunderstandings about

> when an issue is considered solved or not.

Reviewer: We could add another sentence here emphasizing the second-order consequence that solving their own tickets provides relief for support teams. By making support tickets collaborative, problem solving becomes a team activity so issues get understood and solved more quickly. Faster resolutions lead to higher satisfaction for the customer and operational capacity for support teams.

> Configure the feature
> =====================

Reviewer: Too generic
```suggestion
Enable ticket closing
=====================
```

> To configure the feature, go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`,
> click on the team, and click :guilabel:`Edit` and enable :guilabel:`Ticket closing`.

Reviewer: Reads better with time order language and a little more specificity
Wording/flow suggestions:
```suggestion
Get started by navigating to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and
choose the appropriate team. Then click :guilabel:`Edit` and enable :guilabel:`Ticket closing` by
checking the field box.
```

.. image:: close_tickets/close-ticket-settings.png
   :alt: Ticket closing feature in Odoo Helpdesk.

To designate which stage the ticket migrates to once it is closed, navigate to the ticket pipeline
by going to :menuselection:`Helpdesk --> Overview` and clicking :guilabel:`Tickets` on the team's
card.

There are two options: create a new Kanban stage or work with an existing one. For both scenarios,

> click the gear icon next to the stage name, select :guilabel:`Edit Stage`, and enable
> :guilabel:`Closing Stage`.

Reviewer: This is a good start.
What's missing here are guilabels and a little bit more context as to what the reader should be looking for e.g. field types should be specified in the text. As well for v14-15, **Save** needs to be clicked to save the changes.

```suggestion
click the :guilabel:`Settings (gear)` icon next to the stage name, select :guilabel:`Edit Stage`,
and enable :guilabel:`Closing Stage`. After checking the field box, click :guilabel:`Save` to
finish.
```

> If a closing stage is not specified, by default, the ticket is moved to the last stage. If more
> than one stage is set as a closing stage, the ticket is put in the first one.

Reviewer: Be specific, especially when writing a new paragraph. Every paragraph should be able to stand on it's own without relying on surrounding context. Constantly remind the reader where they are and what to look for.

E.g. 
- `is moved to the last stage` --> the last stage, where?
- `is put in the first one` --> the first what?

This would also be better as a **note**.

```suggestion
.. note::
   If a closing stage is not specified, by default, the ticket is moved to the last stage in the
   kanban. If more than one stage is set as a closing stage, the ticket is placed in the first
   closing stage column.
```

> The customer portal
> ===================

Reviewer: This heading isn't needed + is too generic.

The paragraph below it (along with the screenshot) would serve better as a conclusion to the previous section.
Logic: if our support team links customers to this anchor, the reader will have to read the steps above anyway so why not just include it in the previous section?
```suggestion
```

> Now, once the customer logs into their portal, the option :guilabel:`Close this ticket` is
> available.

Reviewer: Reiterate what's going on, especially when starting a new paragraph.

Logic: Connecting paragraphs together with context clues helps readers quickly identify what's being discussed so they can make informed decisions as to whether or not the content is relevant to them; documentation readers skim content and don't typically follow along in a linear fashion , so it's good practice to constantly remind them what's going on, where they are, or what just happened / is about to happen.

```suggestion
With the ticket closing settings now complete, customers can now view the option to
:guilabel:`Close this ticket` when they log into their portal.
```

.. image:: close_tickets/customer-view-close-ticket.png
   :alt: Customer view of ticket closing in Odoo Helpdesk.

Get reports on tickets closed by customers
==========================================

To analyze the tickets that have been closed by customers, go to :menuselection:`Helpdesk -->

> Reporting --> Tickets`. Then, click on the :guilabel:`Filters` menu and add a :guilabel:`Add Custom
> filter`. Next, set the custom filter parameters to :guilabel:`Closed by partner` and :guilabel:`is
> true`. Finally, click :guilabel:`Apply`.

Reviewer: Add + add is redundant
```suggestion
Reporting --> Tickets`. Then, click on the :guilabel:`Filters` menu and choose :guilabel:`Add
Custom filter`. Next, set the custom filter parameters to :guilabel:`Closed by partner` and
:guilabel:`is true`. Finally, click :guilabel:`Apply`.
```

=====================

To configure the feature go to :menuselection:`Helpdesk --> Settings --> Helpdesk Teams --> Edit`
and enable :guilabel:`Ticket closing`.

.. image:: close_tickets/close-ticket-settings.png

>    :alt: Ticket closing in Odoo Helpdesk

Reviewer: If I remember correctly, `align: center` is a depreciated attribute; we no longer need that line. Going to resolve for now and can address if I'm wrong during DR.

In order to designate to which stage the ticket migrates to once it is closed, go to
:menuselection:`Helpdesk --> Overview --> Tickets`.

.. image:: close_tickets/customer-care-tickets.png
   :alt: Ticket closing in Odoo Helpdesk

> You can either create a new Kanban stage or work with an existing one. For both scenarios, go to

Reviewer 2: ```suggestion
There are two options: create a new Kanban stage or work with an existing one. For both scenarios,
click the gear icon next to the stage name, select :guilabel:`Edit Stage`, and enable
:guilabel:`Closing Stage`.
```

:menuselection:`Helpdesk --> Settings --> Edit Stage` and enable :guilabel:`Closing Stage`.

.. image:: close_tickets/edit-column-closing-stage.png

>    :alt: Ticket closing in Odoo Helpdesk

Reviewer 2: This image and its image directive and alt tag can be deleted.

> If a closing stage is not specified, by default, the ticket is moved to the last stage; contrarily,
> if you have more than one stage set as closing, the ticket is put in the first one.

Reviewer 2: ```suggestion
If a closing stage is not specified, by default, the ticket is moved to the last stage. If more than
one stage is set as a closing stage, the ticket is put in the first one.
```

The customer portal
===================

> Now, once the user logs into their portal, the option :guilabel:`Close this ticket` is available.

Reviewer 2: ```suggestion
Now, once the customer logs into their portal, the option :guilabel:`Close this ticket` is
available.
```

.. image:: close_tickets/customer-view-close-ticket.png

>    :alt: Ticket closing in Odoo Helpdesk

Reviewer 2: I would retake this screenshot. It should be zoomed in more and resized. The "Close this ticket" button is supposed to be the focus of this image, but it's so small that you can barely read it.

Reviewer 2: Remember to keep the image width 768px or below; you can play with the browser window width to try to get the correct sizing. For zoom, keep it in the 110% to 125%.

Get reports on tickets closed by customers
==========================================

> To do an analysis of the tickets that have been closed by customers go to :menuselection:`Helpdesk
> --> Reporting --> Tickets --> Filters --> Add Custom filter --> Closed by partner --> Applied`.

Reviewer 2: ```suggestion
To analyze the tickets that have been closed by customers, go to :menuselection:`Helpdesk -->
Reporting --> Tickets`. Then, click on the :guilabel:`Filters` menu and add a :guilabel:`Add Custom
filter`. Next, set the custom filter parameters to :guilabel:`Closed by partner` and :guilabel:`is
true`. Finally, click :guilabel:`Apply`.
```

.. image:: close_tickets/closed-by-search-filter.png

>    :alt: Reports on Ticket closing in Odoo Helpdesk

Reviewer 2: ```suggestion
   :align: center
   :alt: Filter for tickets closed by customers on Odoo Helpdesk's reporting page.
```

Reviewer: See previous comment
